server/base-classes/transport.mdx

```diff
@@ -168,3 +168,26 @@ def add_event_handler(self, event_name: str, handler)
   Voice Activity Detection analyzer. You can set this to either
   `SileroVADAnalyzer()` or `WebRTCVADAnalyzer()`.
 </ParamField>
+
+#### Smart Turn Detection
```

Comment:
I would name this `User End of Turn Detection` or something like that. It's not just the smart turn. We already have the specific file for that.